bigquery/main.go

```diff
@@ -18,14 +18,19 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"math/big"
 	"os"
+	"regexp"
 
 	"cloud.google.com/go/bigquery"
+	"cloud.google.com/go/civil"
 	"github.com/GoogleCloudPlatform/cloud-build-notifiers/lib/notifiers"
 	log "github.com/golang/glog"
 	cbpb "google.golang.org/genproto/googleapis/devtools/cloudbuild/v1"
 )
 
+var rgp = regexp.MustCompile(".*/.*/.*/(.*)/.*/(.*)")
```

Comment:
FYI: Your regexp will match `///////////////////////` for example I think. LMK if you want to make it more robust, but we could follow a garbage-in-garbage-out principle.